static/index.html

```diff
@@ -27,6 +27,7 @@
 
     <script>
       window.buildbuddyConfig = {{.Config}};
+      require = () => { };
```

Comment:
would this work as `window.require = ...` just to be consistent with the line above it?

Also add a comment explaining what this is working around?

Reply:
Done